manifest.json

```diff
@@ -1,27 +1,34 @@
 {
-  "manifest_version": 2,
-  "name": "Send to Outline",
-  "version": "1.2.0",
+    "manifest_version": 2,
+    "strict_min_version": "67.0",
+    "name": "Send to Outline",
+    "version": "1.3.0",
 
-  "description": "Sends the reader to the same article on https://outline.com",
-  "icons": {
-    "48": "icons/md-arrow-round-forward.svg",
-    "96": "icons/md-arrow-round-forward.svg"
-  },
+    "browser_specific_settings": {
+        "gecko": {
+            "id": "send-to@outline.com"
```

Comment:
I feel a little uncomfortable using an email address I'm not associated with as the id for this extension

What do you think about updating this to a guid as described here? - https://developer.mozilla.org/en-US/docs/Mozilla/Tech/XPCOM/Generating_GUIDs